config.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  auto-delete:
```

Comment:
Do we have a spec for this config option? I don't remember one. 
In general config options go through specs / managers approval

Reply:
It was added to the MongoDB VM Charm as a part of the MongoDB Provider lib, it was added by Mykola over two years ago. Not sure what the spec it would go to.

I can remove it, but then I need to update the mongodb_provider lib on the VM charm side

Reply:
If it was added already added - then we can keep it. We'll need to document this config option in a spec however (not imminent, but let's please create a ticket)